src/github.com/getlantern/flashlight/genconfig/cloud.yaml.tmpl

```diff
@@ -7,7 +7,7 @@ client:
   frontedservers: []
   chainedservers: {{range .fallbacks}}
     fallback-{{.ip}}:
-      addr: {{.ip}}:443
+      addr: {{.ip}}
```

Comment:
So `fallback-{{.ip}}:` will expand to `fallback-x.x.x.x:443:`? The extra colon seems invalid.